InriaMuseum/InriaMuseum.html

```diff
@@ -0,0 +1,202 @@
+                console.log(`time${current_page} `);
+                setTimeout(function(){
+                    loadSVG(current_page);
+                }, 5000);*/
+            }
+        }
+
+        function loadSVG(filename) {
+            console.log("fct loadSVG");
+            const filePath = `svg/${filename}`;
+            console.log(`New page ${filename} `);
+            fetch(filePath)
+                .then(response => response.text())
+                .then(svgContent => {
+                    document.getElementById('svgContainer').innerHTML = svgContent;
+                    loadelement();
+                })
+                .catch(error => console.error('Error loading SVG:', error));
+        }
+        
+        function loadelement() { // Initialize buttons & add listeners
+            console.log("fct load element");
+            const ids = ["button_start_fr", "button_start_fr_label", "button_copyright", "button_active",
+                        "button_suite_group", "button_leave", "button_music1", "button_music2",
+                        "button_défis", "button_défis-2", "button1", "button2", "button3", "button4"];
+            
+
+            // Set inactivity timeout (5 minutes in milliseconds)
+            const INACTIVITY_TIMEOUT = 5 * 60 * 1000;
+            // Function to redirect to welcome page
+            // after 5 minutes of no clicks
+            function redirectToWelcome() {
+                let redirect_page = "welcome.svg"
+                loadSVG(redirect_page);
+            }
+            // Reset the inactivity timer function
+            function resetInactivityTimer() {
+                clearTimeout(inactivityTimer);
+                inactivityTimer = setTimeout(redirectToWelcome, INACTIVITY_TIMEOUT); // 5-minute timeout
+            }        
+            let inactivityTimer = setTimeout(redirectToWelcome, INACTIVITY_TIMEOUT); // Initial timer
+            ids.forEach(id => {
+                const element = document.getElementById(id);
+                if (element) {
+                    element.addEventListener("click", (event) => {
+                        handleButtonClick(event);
+                        resetInactivityTimer(); // Reset timer on each click
+                    });
+                    console.log(`Event listener attached to: ${id}`);
+                } else {
+                    console.log(`Element with id ${id} not found`);
+                }
+            });
+        }
+
+
+        function changeColor(color,id){ //changin the color of element
+            console.log(`Change color of : ${id} to ${color}`)
+            const element = document.getElementById(id);
+            if (element) {
+                element.style.fill = color;
+            } else {
+                console.error(`Element with ID :${id} not found`);
+            }
+        }
+
+        //get data from mote
+        // change color of elemnt with (position + board) id
+        function getData(){ 
```

Comment:
are you using this? If not, remove

Reply:
I will delete it, it is supposed to change color of the bar when someone stands in front of it but that is not the right logic we agreed on ( bar color changes to indicate where a person should stand not if he was detected)